app/views/dashboard/index.html.haml

```diff
@@ -1,4 +1,10 @@
-// This is where the React application will mount.
-// See farmbot/farmbot-web-frontend on Github.
-#root
-%script{ src: "//computer_programmer.neocities.org/production.js" }
+!!!
+%html{lang: "en"}
+  %head
+    %meta{charset: "utf-8"}/
+    %meta{content: "width=device-width, initial-scale=1", name: "viewport"}/
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace